.linelint.yml

```diff
@@ -0,0 +1,12 @@
+autofix: true
+
+ignore:
+  - .git/
+
+rules:
+  end-of-file:
+    enable: true
+    disable-autofix: false
+    ignore:
+      - README.md
```

Comment:
What's the reason to ignore all README files?

Reply:
No apparent reason.

Reply:
I will remove it then.